extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -95,6 +99,80 @@
   border-bottom-right-radius: 50% !important;
 }
 
+.i-amphtml-story-interactive-slider-average-bubble {
```

Comment:
This class could be called sth different since the icon is not really a bubble (I first though the bubble was referring to the `Average answer` bubble but this doesn't seem to be that). We can call it `average-indicator`